VTUL/dlp-access
# Fixes collection tree to allow for expanding and collapsing nodes.

**Fixes collection tree to allow for expanding and collapsing nodes.**
* * *

**JIRA Ticket**: ([link](https://webapps.es.vt.edu/jira/browse/LIBTD-2545)) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Fixes collection tree to allow for expanding and collapsing nodes.

# What's the changes? (:star:)
* Adds click handler to respond to expand/collapse clicks by setting component's "expanded" state with appropriate nodeIDs

# How should this be tested?
* Visit a collection page and click the `+` and `-` icons in the collection tree to check if nodes expand and collapse correctly

# Additional Notes:
* branch: `LIBTD-2545`

# Interested parties
@yinlinchen 

(:star:) Required fields
